relnotes/taskextcli.feature.md

```diff
@@ -25,3 +25,6 @@ class TestApp : CliApp
     }
 }
 ```
+
```

Comment:
I'd put this as a separate note, otherwise `DaemonApp` users will miss it under the heading `CliApp now supports TaskExt`.